cosmos/job/drm/drm_ge.py

```diff
@@ -137,15 +139,15 @@ def _get_task_return_data(self, task):
             system_time=float(d['ru_stime']),
 
             avg_rss_mem=d['ru_ixrss'],
-            max_rss_mem_kb=convert_size_to_kb(d['ru_maxrss']),
+            max_rss_mem_kb=convert_size_to_kb(d['maxrss']),
```

Comment:
`ru_maxrss` is already in kb. And SGE collects rss stats in two places. I switched this to `maxrss` to match `maxvmem`, below

Reply:
nice!